packages/analytics-core/src/core-client.ts

```diff
@@ -75,6 +76,10 @@ export class AmplitudeCore<T extends Config> implements CoreClient<T> {
 
   async dispatch(event: Event) {
     try {
+      // if not init yet, only #push the events, #apply will be skipped in timeline
+      if (!this.config) {
+        return this.timeline.push(event, this.config);
```

Comment:
If config does not exist, do you need to push the event with `this.config`?

Reply:
I'm thinking that `this.config` is a reference the the Client instance property which will later be defined. However, i'm concerned if they instantiate config to `optOut: true` which may not catch.